english/Trip-Yourself-Hue/trippy.md

```diff
@@ -0,0 +1,5 @@
+Choose your poison, get your brains all mooshed up    
+---> [Pony](https://www.youtube.com/watch?v=X0gyqsPdWFE)   
+--> [Horse](https://www.youtube.com/watch?v=QCGcCC065CY)   
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.